File: Program3.c
 //Problem Statement : Accept two values from user and perform the Addition

//Step 1: understand the problem statemnet
//conclusion: we have to accept 2 integers and perform its Addition

//step 2 : write the algorithm
/*
  Accept first number from user and store it into no1
  Accept Second number from user and store it into no2
  create one variable to store the result  

*/

//step 3: Decide the Programming language(C/C++/Java)

//Step 4: Write the Program  

#include<stdio.h>

int main()
{

   int iNo1,iNo2,iAns;
   printf("Enter first number :\n");
   scanf("%d",&iNo1);

   printf("Enter first number :\n");
   scanf("%d",&iNo2);

   iAns = iNo1 + iNo2;

   printf("Additon is : %d\n",iAns);


    return 0;
    
}

File: Java Programs/Problems on Numbers/Program3.java

//Addition of two number
//By Accepting input by using buffered class 
import java.io.*;

class Program3
{
    public static void main(String Args[]) throws Exception
    {
      BufferedReader bobj = new BufferedReader(new InputStreamReader(System.in));
      
      int iNo1 =0, iNo2 = 0;
      int iAns = 0;

      System.out.println("Enter First Number  :");
      iNo1 = Integer.parseInt(bobj.readLine());
      
      System.out.println("Enter First Number  :");
      iNo2 = Integer.parseInt(bobj.readLine());
      
      iAns = iNo1 + iNo2;

      System.out.println("Addition of two numbers is :"+iAns);

  }

}